gstools/krige/base.py

```diff
@@ -188,8 +213,8 @@ def __call__(
         field = np.empty(pnt_cnt, dtype=np.double)
         krige_var = np.empty(pnt_cnt, dtype=np.double) if return_var else None
         # set constant mean if present and wanted
-        if only_mean and self.has_const_mean:
-            field[...] = self.get_mean() - self.mean  # mean is added later
+        if only_mean and self.drift_no == 0:
+            field[...] = self.get_mean(post_process=False)
         # execute the kriging routine
         else:
             # set chunk size
```

Comment:
These comments are superfluous.

Reply:
Wanted to be rather explicit about what is done in each line. will keep it for now.